### ISSUE TITLE
[Bug]: sp-table default role should be table not grid

### ISSUE THREAD

**Issue by carolframpton** [adobe/spectrum-web-components#4955]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-table

### Expected behavior

Given the default role for sp-table is role="grid" I expect the component to support the keyboard navigation associated with the grid role.

ref: https://www.w3.org/WAI/ARIA/apg/patterns/grid/ see Keyboard Interaction For Data Grids section

Per @jnurthen, the default role should be table.

### Actual behavior

Navigation between sortable column headers and cell which contain inputs is with tabs.

Yes, I can manually set role=table.

### What browsers are you seeing the problem in?

Chrome

### How can we reproduce this issue?

1. Go to (https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/table-virtualized--virtualized)
2. Click on storybook header and tab until you get to the first column header, tab to the 2nd column header. Left arrow to go back to the first column header.

### Sample code or abstract reproduction which illustrates the problem

https://opensource.adobe.com/spectrum-web-components/storybook/index.html?path=/story/table-virtualized--virtualized

### Severity

None

**Comment by najikahalsema** [comment]
Closing as a duplicate of #3395
